9.4/paper.bib

```diff
@@ -613,6 +620,32 @@ @article{MUMPS:2
    pages   = {136-156}
 }
 
+@article{MUMPS:3,
+  title = {{Performance and Scalability of the Block Low-Rank Multifrontal
+  Factorization on Multicore Architectures}},
+  author = {P.R. Amestoy and A. Buttari and J.-Y. L'Excellent and T. Mary},
+  journal = {ACM Transactions on Mathematical Software},
+  volume = {45},
+  issue = {1},
+  pages = {2:1--2:26},
+  year={2019},
+}
+
+See: https://ginkgo-project.github.io/ginkgo/doc/develop/citing_ginkgo.html
+@article{Anzt2022,
+  doi = {10.1145/3480935},
+  url = {https://doi.org/10.1145/3480935},
+  year = {2022},
+  month = mar,
+  publisher = {Association for Computing Machinery ({ACM})},
+  volume = {48},
+  number = {1},
+  pages = {1--33},
+  author = {Hartwig Anzt and Terry Cojean and Goran Flegar and Fritz G\"{o}bel and Thomas Gr\"{u}tzmacher and Pratik Nayak and Tobias Ribizel and Yuhsiang Mike Tsai and Enrique S. Quintana-Ort{\'{\i}}},
```

Comment:
We try to abbreviate first names, to be consistent. I don't know whether the bib style already does this, but if not, please do the abbreviations.

Reply:
The bibstyle uses abbreviations.
![image](https://user-images.githubusercontent.com/18285973/175476390-70de9bf6-ff9e-4d67-935a-6a7de266bc5e.png)